src/components/PieCharts/Donut/DonutView.js

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react';
```

Comment:
Can you create the stories similar to https://github.com/CBIIT/bento-tools/blob/master/src/stories/index.js#L37 for both donut and sunburst

Reply:
sure